code-scanning/properties/inclusiveness-analyser.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Inclusiveness Analyzer",
+    "creator": "Microsoft",
+    "description": "The Inclusiveness Analyzer is a GitHub action that checks your repository for offensive / exclusive terms.It also provides context on why a word is exclusive and suggests alternate terms that can be used instead.",
+    "iconName": "inclusiveness-analyser",
+    "categories": [ "Code Scanning"]
```

Comment:
This doesn't seem to belong to `Code Scanning` category. This category is usually for scanning code for security related issues.

Reply:
Seems more in automation

Reply:
@Phantsure thanks for the feedback! Inclusiveness analysis may be an oddball. I moved it to automation.